build/cmake/terrame/CMakeLists.txt

```diff
@@ -382,28 +381,35 @@ if (APPLE)
            )
 
     set(TERRAME_LUA_SRC_DIR ${TERRAME_SRC_DIR}/lua)
+
     install(DIRECTORY ${TERRAME_LUA_SRC_DIR}
             DESTINATION ${TERRAME_CURRENT_BINARY_DIR}
             COMPONENT runtime
            )
 
     set(TERRAME_PACKAGES_DIR ${TERRAME_ABSOLUTE_ROOT_DIR}/packages)
+
 	if (NOT TERRAME_CREATE_INSTALLER)
```

Comment:
A good programming practice is not to use `if not` when there is an `else`. Put the positive part first and leave the negative part to the `else`. Update this (and possibly other) if while reviewing indentation #2044.